### PR TITLE
WIP: Reformating stuff

### PR DESCRIPTION
**Pull request by manuelliebchen**

### CODE REVIEW

**Comment by johannwagner** [comment]
I will wait for an dedicated comment to merge this into Jasper-Ben-master.

**Comment by johannwagner** [comment]
You also need to rebase your branch to the current Jasper-Ben-master due to some hotfixes from @Jasper-Ben

File: MerkzettelMathe3.tex [hunk 1]
```diff
@@ -398,30 +414,23 @@
...
+					\item Nach y umstellen
 				\end{compactenum}	
 			\subsubsection{Variation der Konstanten (inhomogen)}
 				Wenden wir an wenn wir die Variablen nicht geteilt bekommen:\\
 				$y'+b(x)\cdot y=0$ für $y_1$\\
 				$y'+b(x)\cdot y=a(x)$ für $y_2$\\
 				
 				$y_1 = c\cdot e^{\int b(x)dx}$\\
-				$y_2 = \int(a(x)e^{\int b(x)dx})dx\cdot e^{-\int b(x)dx}$\\
+				$y_2 = \int(a(x)e^{-\int b(x)dx})dx\cdot e^{-\int b(x)dx}$\\
```
Comment (Jasper-Ben) [inline MerkzettelMathe3.tex]: AFAIK this is not correct. This line should stay the same and the "-" should be added in y1, as done in my hotfix PR.

File: MerkzettelMathe3.tex [hunk 2]
```diff
@@ -398,30 +414,23 @@
...
-		\subsection{Matrixnormen}
-			\begin{tiny}
-				\begin{align*}
-					\left|\bspVec\right| &= \sqrt{x_1^2 + ... + x_n^2}\\
-					&... 
-				\end{align*}
-			\end{tiny}
 	\section{Differentialgleichung}
 		\subsection{DGL 1. Ordnung}
 			\subsection{homogen}
```
Comment (Jasper-Ben) [inline MerkzettelMathe3.tex]: In my PR I completely removed this section, as you can (to my knowledge) just use y1 (y=y1 for uniform) from the next subsection.

File: MerkzettelMathe3.tex (hunk 1 above)
Comment (Jasper-Ben) [inline MerkzettelMathe3.tex]: AFAIK this is not correct. This line should stay the same and the "-" should be added in y1, as done in my hotfix PR.

File: MerkzettelMathe3.tex [hunk 3]
```diff
@@ -122,52 +163,28 @@
...
-		    	P(A_1 \cap A_2) &= P(A_1) \cdot P(A_2), A_1 \cap A_2 = \emptyset \\
-		    	P(A_1 \cup A_2) &= P(A_1) + P(A_2), A_1 \cap A_2 = \emptyset \\
-		    	P(\Omega \backslash A) &= 1 - P(A)\\
 		    	P(\emptyset) &= 0\\
-		    	A \subseteq B &\iff P(A) \leq P(B) \\
-		    	P(A|B) &= \frac{P(A \cap B)}{P(B)} \\
-		    	&= \frac{P(B|A)\cdot P(A)}{P(B)} \\
+		    	P(\Omega) &= 1\\
+		    	P(A \cup B) &= P(A) + P(B)-P(A \cap B)\\
+		    	P(A \cap B) &= P(A) \cdot P(B)\\
```
Comment (Jasper-Ben) [inline MerkzettelMathe3.tex:182]: AFAIK this is only the case, if A and B are independent? This should be clarified.

**Review by Jasper-Ben — COMMENTED** [review]
Please check :) @manuelliebchen

**Comment by Jasper-Ben** [comment]
Also we need to do something about the cut off table in 4.7

**Comment by manuelliebchen** [comment]
> Also we need to do something about the cut off table in 4.7

I removed the table after moving the cdf and variancy to the sections of the distrubution

**Comment by Jasper-Ben** [comment]
> > Also we need to do something about the cut off table in 4.7
> 
> I removed the table after moving the cdf and variancy to the sections of the distrubution

Alright :) Btw, top right corner on page 1 is currently cut off.

**Comment by manuelliebchen** [comment]
So, I finished and you can Merge a finale time.

**Pull request merged**